docs/keywords.md

```diff
@@ -0,0 +1,56 @@
+  # NL-Augmenter Keywords
+
+Every transformation should have keywords that identify the type of transformation that is being written. The following list can be used as reference.
+- the type of linguistic change that the transformation is attempting eg. morphological, lexical, syntactic, word-order, discourse, noise etc.
+- the type of algorithm that is written eg. rule-based, model-based, api-based, external-knowledge-based, transformer-based, parser-based, etc.
+- the naturalness of the generation eg. natural-sounding, natural-looking, unnatural, etc.
+- the potential accuracy & precision of the generation eg. accurate, near-accurate, high-precision, high-coverage, etc.
```

Comment:
I don't know what accuracy and precision mean in this context. I would elaborate and/or rephrase.

Does accurate mean that the transformation preserves the original meaning of the string? Does precise mean that if the transformation changes the meaning, it does so in a consistent way across multiple strings (eg, it changes names or numbers consistently)?

The same comment applies to the "**The potential accuracy & precision of the generation** " section below.

Reply:
Thanks for the clarity. Accuracy was intended to give a measure of how well the transformation generates what it intends to generate - and yes in the case of sentence operation- it would imply preserving meaning. And since I've merged precision and accuracy both now, I will leave out precision (I initially saw precision aligned with your definition to cater to rule-based approaches)  

I have updated that. Happy to revise it if it looks confusing.